adapters/multiply.go

```diff
@@ -3,14 +3,15 @@ package adapters
 import (
 	"fmt"
 	"math/big"
+	"strconv"
 
 	"github.com/smartcontractkit/chainlink/store"
 	"github.com/smartcontractkit/chainlink/store/models"
 )
 
 // Multiply holds the a number to multiply the given value by.
 type Multiply struct {
-	Times int64 `json:"times"`
+	Times interface{} `json:"times"`
```

Comment:
After further thought, I think we should stay away from `interface{}` when possible, and place this logic in `UnmarshalJson`. Similar to what geth does in their `hexutil` package: https://github.com/ethereum/go-ethereum/blob/master/common/hexutil/json.go#L192

A simpler version of that could look like this:

common.go

```golang
Times JSONFloat `json:"times"`
...

type JSONFloat float

// UnmarshalJSON implements json.Unmarshaler.
func (jf *JSONFloat) UnmarshalJSON(input []byte) error {
        // case statement w timesToFloat
        ...
	*jf = safeFloat
}
```

Feel free to change the names, but you get the idea. This way we have a strong type instead of `interface{}` that encapsulates the deserialization.

Reply:
Good point! At first I thought that it'll require additional unmarshaling when `Multiply` is initialized, but then I saw that it's already done: https://github.com/smartcontractkit/chainlink/blob/master/adapters/adapter.go#L41